google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/readrows/ReadRowsConvertExceptionCallable.java

```diff
@@ -78,7 +78,7 @@ private Throwable convertException(Throwable t) {
     // Long lived connections sometimes are disconnected via an RST frame. This error is
     // transient and should be retried.
     if (t instanceof InternalException) {
-      if (t.getMessage() != null && t.getMessage().contains("Received Rst stream")) {
+      if (t.getMessage() != null && t.getMessage().contains("Received Rst Stream")) {
```

Comment:
maybe we use a case insensitive comparison?

Reply:
I would just simplify this to lowercase().contains("rst_stream") || lowercase.contains("rst stream")